lore/species/tajaran/The Great Frost.md

```diff
@@ -0,0 +1,85 @@
+
+The Shards of Messa
+The growth in numbers the Shards of Messa once had has weakened with every year as people grew less and less desperate for answers. The early sabotage and violence they created began to ebb as the witch hunts began. Though some still remain, hidden and waiting, within the holds these maddened cultist no longer have the power and fear they once held. This external strife has made the remaining cultist experts of politics and subterfuge, their numbers dwindled to only the most ruthless and cunning. Still, with cult activity nearly gone many choose to believe the Shards of Messa are entirely eradicated. Yet the odd murder every few years where someone is found dead, their throat slit by a glass shard, makes those in power worry they still remain.
+
+The Legend of Akari Tal�Wrack
+
+Akari Tal�Wrack has become a mild enigma among the tajaran people. What is known of his past life for sure was he was once a brilliant and wealthy scientist and that he created the first hearth engines. Hailing from the Siik his life was largely unknown until the great frost event. Many say that Akari survived the great frost and moved from hold to hold in disguise to aid them by fixing their hearth engines in secret. Wild speculation and rumor became so inherent when discussing the life of Akari that no one is able to say for certain what he had done or if he had survived at all. Some groups believe he was behind the creation of the first frost breakers, other say the glare of S�randarr was his creation gifted unto the holds. Even wilder speculation say that he was the prophet of S�randarr himself sent to save the tajaran people from Messa. A few even claim that Akri was S�randarr. Whatever is true he has since fallen into legend and many expeditions to search ruins for religious artifacts check each body found to see if they bear the tell tale sign of Akari�s supposedly strange facial birthmark, a gray mark in the shape of a perfect ring.
+
+In any case the tremors of his actions echo through tajaran society today. In most holds money commonly bears either the depiction of his face or the symbol of his birthmark above or on the face of rulers. Many temples have small shrines devoted towards the savior of the tajaran and it is not uncommon for prayers to thank Akari for his actions. The greatest shrines devoted towards him are within the hearth engines and frost breakers themselves, nearly all are decorated with his symbol and treated as holy objects by some.

```

Comment:
"In any case, the tremors..."
"In most holds, money commonly..."
"... frost breakers themselves, nearly all of which are decorated..."